src/Digdir.Domain.Dialogporten.Infrastructure/Persistence/Sql/Configuration/WriteHeavyTablesAutoVacuum.sql

```diff
@@ -0,0 +1,61 @@
+-- Override autovacuum settings for large tables
+
+DO $$
+DECLARE
+
+  vac_scale     CONSTANT numeric := 0.0001;  -- 0.01%
```

Comment:
This might be to aggressive, as combined with high `cost_limit` causes the autovacuum processes to use a substantial amount of time on the largest tables (and not being able to frequently vacuum the MassTransit tables)